README.md

```diff
@@ -10,7 +10,7 @@ The [Eggplant DAI](https://www.eggplantsoftware.com/digital-automation-intellige
 
 **Step 1:** Login to your Jenkins
 
-Go to http://localhost:{portnumber}/ and login into your Jenkins account
+Go to `http://dai_server_hostname:portnumber/` and login into your Jenkins account
```

Comment:
I think this is the Jenkins URL, not the DAI server URL.

Perhaps, we can just mention : Go to your Jenkins home page and login your Jenkins account